sumologic/sumologic_client.go

```diff
@@ -60,6 +61,13 @@ func createNewRequest(method, url string, body io.Reader, accessID string, acces
 	return req, nil
 }
 
+func logRequestAndResponse(req *http.Request, resp *http.Response) {
+	var maskedHeader = req.Header.Clone()
+	maskedHeader.Set("Authorization", "xxxxxxxxxxx")
+	log.Printf("[DEBUG] Request made to Sumo Logic: [Method=%s] [URL=%s] [Headers=%s]", req.Method, req.URL, maskedHeader)
```

Comment:
Change the log to "Sent request: "? URL is enough to identify where we are making the request to.

Needs a "\n" at the end of log.